specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json

```diff
@@ -7841,6 +7841,10 @@
         "endCharIndex": {
           "description": "The index within the utterance where the extracted entity ends.",
           "type": "integer"
+        },
+        "role": {
+          "description": "The role.",
```

Comment:
If possible, better be something more informative.

Reply:
Addressed.